src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -22,7 +22,7 @@
   public static final String DEFAULT_HOST = "localhost";
   public static final int DEFAULT_PORT = 6379;
   public static final int DEFAULT_SENTINEL_PORT = 26379;
-  public static final int DEFAULT_TIMEOUT = 2000;
+  public static final int DEFAULT_TIMEOUT = 200;
```

Comment:
?? Change the default timeout configuration used in Jedis overall?. This will definitely won't happen